docs/gallery/interactive/pyfar_audio_objects.ipynb

```diff
@@ -0,0 +1,413 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.signal_type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**DFT Normlaization:** For *power* Signals, different DFT normalization can be used that differently scale the values stored in `signal.freq`. For *energy* signals, the DFT normalization is always 'none'. For more information refer the `Fast Fourier Transform` examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.fft_norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Signal cshape, length, and caxis\n",
+    "\n",
+    "Signals can contain multiple channels of audio data and the shape of the data inside the audio object is often important. There multiple attributes describing this.\n",
+    "\n",
+    "**Channel shape:** The *channel shape*, in short `cshape` gives the shape of the data inside an audio object but ignores the number of samples or frequency bins. For example our first Signal created above has one channel and 4 samples, hence the data is of chape `(1, 4)`. Because the `cshape` ignores the samples it is `(1, )`. There are two ideas behind this. First, digital signal processing methods often work on channels, and second the length of the signal might be different in the time and frequency domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.cshape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Length:** The signal length in samples and bins is stored in the properties"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.n_samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.n_bins"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that only the half sided spectrum is stored for real valued time signals. Hence the length differs in the time and frequency domain in this case."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Channel axis:** Some functions in pyfar operate along one or more axes of the data. In analogy, to the channel chape, these axes are referred to as *channel axis* or in short `caxis`. Consider a signal of `cshape=(3, 4)` and a duration of `n_samples=128`. In this case `caxis=0` referst to the first dimension of size 3 and `caxis=-1` referns to the last dimension of size 4 but *not* dimension containing the 128 samples."
```

Comment:
```suggestion
    "**Channel axis:** Some functions in pyfar operate along one or more axes of the data. In analogy, to the channel shape, these axes are referred to as *channel axis* or in short `caxis`. Consider a signal of `cshape=(3, 4)` and a duration of `n_samples=128`. In this case `caxis=0` refers to the first dimension of size 3 and `caxis=-1` refers to the last dimension of size 4 but *not* the dimension containing the 128 samples."
```
I feel like this might be a bit confusing.